panel/src/components/Forms/Input/SelectInput.vue

```diff
@@ -70,10 +63,6 @@ export default {
 			return this.placeholder ?? "—";
 		},
 		hasEmptyOption() {
-			if (this.empty === false) {
-				return false;
-			}
-
 			return !(this.required && this.default);
```

Comment:
To make it easier to understand what this does, we could split this up and add a comment:

```suggestion
			// empty option is only hidden if the field is both required and has a default
			return !this.required || !this.default;
```

Reply:
I like it, but I think now the comment is confusing as it doesn't quite describe what the code below expresses (even if the result is the same). Maybe keep the old notation with the comment?

Reply:
You are right. I first changed the logic because it twisted my head and then wrote the comment, but the comment is much clearer with the original syntax.